File: ServiceMap/src/ServiceMap/wwwroot/app/servicesTnt/serviceTnt.ts
import { IPage } from '../pagination/page';
import { IResult } from '../shared/common';

export interface IServiceTnt {
    depotCode: string;
    town: string;
    fromPostcode: string;
    toPostcode: string;
    sobota: boolean;
    eX9: boolean;
    eX10: boolean;
    eX12: boolean;
    priority: string;
    wieczorneDostarczenie?: boolean;
    standardDeliveryOd: string;
    standardDeliveryDo: string;
    pickUpDomesticZgl: string;
    dateTimePickUpEksportSmZgl: string;
    samochodZwindaDostepnyWstandardzie?: boolean;
    diplomatNextDay: string;
    serwisMiejski?: boolean;
    serwisPodmiejski?: boolean;
    pickUpDomesticCzas: string;
    pickUpEksportSmCzas: string;
}
export interface IServiceFilter {
    postCode: string;
    cityName: string;
}

export interface IServiceTntResult {
    serviceTnt: IServiceTnt[];
    paging: IPage;
    requestsPerDay: IRequestsPerDay;
    result: IResult;
}

export interface IRequestsPerDay {
    limitOfRequestsPerDay?: number;
    numberOfRequestsPerDay?: number;
}

File: ServiceMap/src/ServiceMap/Models/Service_Tnt/ServiceTnt.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ServiceMap.Models.Service_Tnt
{
    public class ServiceTnt
    {
        public string DepotCode { get; set; }
        public string Town { get; set; }
        public string FromPostcode { get; set; }
        public string ToPostcode { get; set; }
        public bool Sobota { get; set; }
        public bool EX9 { get; set; }
        public bool EX10 { get; set; }
        public bool EX12 { get; set; }
        public string Priority { get; set; }
        public bool? WieczorneDostarczenie { get; set; }
        public string StandardDeliveryOd { get; set; }
        public string StandardDeliveryDo { get; set; }
        public string PickUpDomesticZgl { get; set; }
        public string DateTimePickUpEksportSmZgl { get; set; }
        public bool? SamochodZwindaDostepnyWstandardzie { get; set; }
        public string DiplomatNextDay { get; set; }
        public bool? SerwisMiejski { get; set; }
        public bool? SerwisPodmiejski { get; set; }
        public Single? PickUpDomesticCzas { get; set; }
        public Single? PickUpEksportSmCzas { get; set; }
        public long RowNumber { get; set; }
        public int TotalCount { get; set; }
    }


    public class Test
    {
        public string DepotCode { get; set; }
        public string Town { get; set; }
        public string FromPostcode { get; set; }
        public string ToPostcode { get; set; }
      

    }
}

